src/trix/models/composition.coffee

```diff
@@ -252,7 +252,12 @@ class Trix.Composition extends Trix.BasicObject
       @setCurrentAttribute(attribute)
 
   decreaseBlockAttributeLevel: ->
-    if attribute = @getBlock()?.getLastAttribute()
+    if @getBlock()?.isTerminalBlock()
```

Comment:
L257-L259 decrease the block attribute level when there is a terminal block.

Currently, terminal blocks disable all buttons other than the terminal attribute button (ie: heading). To unindent a list containing a heading, you can *only* use the unindent button -- the list button will be disabled. This is by design and only allows the second to last block attribute to be removed. In other words, if a block's attributes are:

```
[bulletList, bullet, quote, heading]
```

the list button will be disabled, and unindenting will remove `quote`. This seemed like a reasonable middle ground between:

- Only being able to remove the last attribute (ie: the functionality before this commit)
- Being able to remove any attribute once a terminal attribute has been applied

On the other hand, if your block contains `[bulletList, bullet, quote, heading]` it means you need to unindent/remove `quote` before you can unindent/remove `bullet` Thoughts on this?